src/app/shell/About.tsx

```diff
@@ -37,6 +38,14 @@ const redditLink = 'https://destinyitemmanager.reddit.com';
 const discordLink = 'https://discord.gg/UK2GWC7';
 const wikiLink = 'https://github.com/DestinyItemManager/DIM/wiki';
 
+const systemInfo = getSystemInfo();
+
+function getSystemInfo() {
+  const parser = new UAParser();
+  const info = `${parser.getBrowser().name} ${parser.getBrowser().version} - ${parser.getOS().name} ${parser.getOS().version}`;
```

Comment:
Every time you call `getBrowser()` or `getOS()`, the user agent gets parsed again.

A more efficient way of doing it would be something like this:

```suggestion
  const { name: browserName, version: browserVersion } = parser.getBrowser();
  const { name: osName, version: osVersion } = parser.getOS();
  const info = `${browserName} ${browserVersion} - ${osName} ${osVersion}`;
```